File: strings/count.js


const counter = str => {
    let chars = str.match(/[a-zA-Z]/ig).length;
    let digits = str.match(/[0-9]/g).length;
    let syms = str.length - (chars + digits);

    return `Total counts of chars, digits, and symbols\n\nChars = ${chars}\nDigits = ${digits}\nSymbols = ${syms}`;
}


let str1 = "P@#yn26at^&i5ve";

console.log(counter(str1));

File: strings/count.py
import re 

def counter(s):
    
    letters = len(re.findall('[a-zA-Z]', s))
    nums = len(re.findall('\d+', s))
    symbols = len(s) - (letters + nums)

    print("Total counts of chars, digits, and symbols\n")
    print("Chars = {}".format(letters))
    print("Digits = {}".format(nums))
    print("Symbol = {}".format(symbols)) 


str1 = "P@#yn26at^&i5ve"

counter(str1)